### ISSUE TITLE
All statistics are 0

### ISSUE THREAD

**Issue by Zanejins** [jstrieb/github-stats#89]
I creating a personal access token. Here are the details about it. 
![image](https://user-images.githubusercontent.com/98217021/218635204-ffa7836e-2df0-4f07-aa8a-fd5039dfdc08.png).

But all statistics are 0. I am confused. 
![image](https://user-images.githubusercontent.com/98217021/218636041-cfc6c32d-1345-4212-8bec-478b6e338607.png)

**Comment by Zanejins** [comment]
I know the reason. the token should be Tokens(Classic)  instead of Fine-grained tokens.
The description of [[README.md]](https://github.com/jstrieb/github-stats/blob/master/README.md)  may be incorrect.
My reference is [[issue81]](https://github.com/jstrieb/github-stats/issues/81)

**Comment by py563** [comment]
yes, we need to update the readme with only use tokens(classic). I have the fine-grained token with read repo properties still as you mentioned it generates images with 0 count

**Comment by jstrieb** [comment]
Addressed by 6f25f02f82141f3bc35bba4a92feb590e6bbcca9.
